### PR TITLE
support studio command

### LINKED ISSUE #2: Support quick start nebula studio

### PR DESCRIPTION
**Pull request by sixwaaaay**
This PR adds studio command to deploy nebula graph studio.

The changes include:

- Implemented the studio command to facilitate the deployment of studio components and remove the deployment created by the command.
- Created tests for the command and configured a GitHub Actions workflow to automatically run the tests.

fix #2

### CODE REVIEW

File: cmd/root.go [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+	"path/filepath"
+)
+
+// Flag Values for RootCmd
+var (
+	kubeConfig string
+)
+
+var RootCmd = &cobra.Command{
+	Use:   "operator-cli [command]",
```
Comment (kqzh) [inline cmd/root.go]: maybe we  can make naming more concise, like ngctl

File: cmd/root.go [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+	"k8s.io/client-go/util/homedir"
+	"path/filepath"
```
Comment (kqzh) [inline cmd/root.go]: use goimports to sorting and classifying dependent packages

File: cmd/studio.go [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+				return err
+			}
+			log.Printf("Deployment created: %s\n", deployment.Name)
+			service := studio.CreateService(name, labels, nodePort)
+			service, err = clientSet.CoreV1().
+				Services(namespace).
+				Create(ctx, service, metav1.CreateOptions{})
+			if err != nil {
+				return err
+			}
```
Comment (kqzh) [inline cmd/studio.go]: ignore already exist err?

File: cmd/studio.go [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+			log.Printf("Deployment deleted: %s\n", name)
+
+			// remove service
+			services := clientSet.CoreV1().Services(namespace)
+			err = util.RemoveResource[corev1.Service](ctx, services, name, label)
+			if err != nil {
+				return err
+			}
```
Comment (kqzh) [inline cmd/studio.go]: ignore not found err?

File: cmd/studio.go [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+		Use:   "install",
+		Short: "install nebula graph studio",
+		Long:  "install nebula graph studio.",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			clientSet, err := util.NewClientSet(kubeConfig)
+			if err != nil {
+				return err
+			}
+			labels := map[string]string{
+				"nebula-studio/operator-cli": name,
```
Comment (kqzh) [inline cmd/studio.go]: label use const is better?

File: internal/studio/studio.go [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+)
+
+// CreateService creates a service for nebula graph studio
+func CreateService(name string, labels map[string]string, nodePort int32) *corev1.Service {
+	return &corev1.Service{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:   name,
+			Labels: labels, // mark the service is created by operator-cli
```
Comment (kqzh) [inline internal/studio/studio.go]: need pass namespace?

File: internal/studio/studio.go [hunk 7]
```diff
@@ -0,0 +1,116 @@
...
+			},
+		},
+	}
+}
+
+// CreateDeployment creates a deployment for nebula graph studio
+func CreateDeployment(name string, labels map[string]string, replicas int32, image string) *appsv1.Deployment {
+	deployment := &appsv1.Deployment{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:   name,
```
Comment (kqzh) [inline internal/studio/studio.go]: need pass namespace?

File: internal/studio/studio.go (hunk 6 above)
Comment (sixwaaaay) [reply internal/studio/studio.go]: namespace was specified as follow:
```go
CoreV1().Services(namespace)
```

**Pull request merged**